sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -876,14 +877,14 @@ def validate(self, validator):
 
     return errors
 
-  def lookup_dataflow_service_option(self, key, default=None):
+  def get_cloud_profiler_service_name(self, key):
```

Comment:
given that we are looking for a specific option, `key` no longer needs to be a parameter. _ENABLE_GOOGLE_CLOUD_PROFILER  is the `key` .